code/game/objects/items/weapons/teleprod.dm

```diff
@@ -13,7 +13,7 @@
 	if(HAS_TRAIT(user, TRAIT_CLUMSY) && prob(50))
 		user.visible_message("<span class='danger'>[user] accidentally hits [user.p_them()]self with [src]!</span>",
 			"<span class='userdanger'>You accidentally hit yourself with [src]!</span>")
-		user.Weaken(stunforce * 3)
+		//user.KnockDown(knockdown_duration * 3)
```

Comment:
Same here. Don't forget to remove the comment

Reply:
the ..() already handles this. so I will remove it